.github/workflows/ci.yml

```diff
@@ -136,7 +136,7 @@ jobs:
         run: pip install halmos
 
       - name: Run halmos
-        run: halmos --error-unknown --test-parallel --solver-parallel --storage-layout=generic --solver-timeout-assertion 0
+        run: halmos --solver-timeout-assertion 0
```

Comment:
- `--error-unknown` is no longer available.
- `--test-parallel` will be deprecated in the next release.
- `--solver-parallel` is deprecated, as parallel solving is now enabled by default.
- `--storage-layout=generic` incurs performance overhead, and is unnecessary for the current tests.